docs/modules/ROOT/pages/components.adoc

```diff
@@ -344,6 +344,91 @@ mod MyContract {
 WARNING: Customizing implementations and accessing component storage can potentially corrupt the state, bypass security checks, and undermine the component logic.
 *Exercise extreme caution*. See {security}.
 
+=== Hooks
+
+:hooks-post: https://fleming-andrew.medium.com/extending-cairo-contracts-with-hooks-c3ca21d1d6b8[Extending Cairo Contracts with Hooks]
```

Comment:
I haven't asked yet, what is the expected way for declaring links: at the top of a page or at the beginning of the sections the link is used in?

Reply:
It depends if it is local or if it's going to be used in different sections, but in reality at least I haven't been following the rule but adding the links where I feel they fit the best.